src/chart-configuration/range-timeseries-data-configuration.ts

```diff
@@ -0,0 +1,162 @@
+export const RANGEDATA_GROUPS_CONFIGURATION = [
+  {
+    type: "hosp_need",
+    ranges: [
+      {
+        lower: {
+          columnName: "hosp_need_2.5",
+          id: "2.5%",
+        },
+        upper: {
+          columnName: "hosp_need_97.5",
+          id: "97.5%",
+        },
+        average: {
+          columnName: "hosp_need_50",
+          id: "50%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+      {
+        lower: {
+          columnName: "hosp_need_25",
+          id: "25%",
+        },
+        upper: {
+          columnName: "hosp_need_75",
+          id: "75%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+    ],
+    chartingMetadata: {
+      title: "Hospital beds needed",
+      xAxisLabel: "Dates",
+      yAxisLabel: "Number of beds (logarithmic scale)",
+    },
+  },
+  {
+    type: "vent_need",
+    ranges: [
+      {
+        lower: {
+          columnName: "vent_need_2.5",
+          id: "2.5%",
+        },
+        upper: {
+          columnName: "vent_need_97.5",
+          id: "97.5%",
+        },
+        average: {
+          columnName: "vent_need_50",
+          id: "50%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+      {
+        lower: {
+          columnName: "vent_need_25",
+          id: "25%",
+        },
+        upper: {
+          columnName: "vent_need_75",
+          id: "75%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+    ],
+    chartingMetadata: {
+      title: "Ventilators beds needed",
+      xAxisLabel: "Dates",
+      yAxisLabel: "Number of ventilators (logarithmic scale)",
+    },
+  },
+  {
+    type: "ICU_need",
+    ranges: [
+      {
+        lower: {
+          columnName: "ICU_need_2.5",
+          id: "2.5%",
+        },
+        upper: {
+          columnName: "ICU_need_97.5",
+          id: "97.5%",
+        },
+        average: {
+          columnName: "ICU_need_50",
+          id: "50%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+      {
+        lower: {
+          columnName: "ICU_need_25",
+          id: "25%",
+        },
+        upper: {
+          columnName: "ICU_need_75",
+          id: "75%",
+        },
+        timeseries: {
+          columnName: "Date",
+        },
+      },
+    ],
+    chartingMetadata: {
+      title: "ICU's needed",
```

Comment:
any title or label should be controled by front-end. The backend just need to pass some string code name so that the front-end and map the code name to whatever title/label it wants to.